1610520/es_tcid.py

```diff
@@ -0,0 +1,68 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+Spits out crash ids for all crash reports in Elasticsearch that have a
+TelemetryClientId field.
+
+Usage:
+
+    python 1610520/es_tcid.py > crashids.txt
```

Comment:
I'd prefer a path like ``scripts/es_tcid_1610520.py``. I'm guessing you intend to delete the script in a week or so, which is why the path is like it is.

Reply:
This brings back scars of finding and deleting scripts that had no documentation and no clear reason what they were doing or why or what they fixed. It took a long time to remove them. They were everywhere in the codebase. We probably still have a bunch.

Thus my scars compelled me to make sure these scripts were clearly connected to that bug and in a directory by themselves and not mixed in with everything else.

How about `scripts/1610520/es_tcid.py`? Would that be ok?

Reply:
That's better, if you intend to keep the script around forever.